SDKBuildScripts/xplat_BuildAndTestAndroid.sh

```diff
@@ -6,16 +6,6 @@ popd
 apkPath=$AndroidProjectPath/app/build/outputs/apk/debug/app-debug.apk
 testAssemblyDir="$1"
 
-Usage="./xplat_BuildAndTestAndroid.sh <path to test assemblies>"
```

Comment:
why not check parameters?

Reply:
I'm not against it, but it's not being invoked, so it's just dead code as it is now.